doc/rados/deployment/preflight-checklist.rst

```diff
@@ -90,7 +90,7 @@ Install ceph-deploy
 To install ``ceph-deploy``, execute the following:: 
 
 	wget -q -O- 'https://download.ceph.com/keys/release.asc' | sudo apt-key add -
-	echo deb http://ceph.com/debian-dumpling/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
+	echo deb http://ceph.com/debian-mimic/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
```

Comment:
This is not entirely correct :( Since it is going to the `master` branch this should use nautilus and the correct url is:

```suggestion
	echo deb https://download.ceph.com/debian-nautilus/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
```

Reply:
Thanks for the review and correction @alfredodeza, will fix shortly 👍